tests/xtro-sharpie/watchOS-SceneKit.ignore

```diff
@@ -17,3 +17,7 @@
 !missing-null-allowed! 'SceneKit.SCNShape SceneKit.SCNShape::Create(UIKit.UIBezierPath,System.nfloat)' is missing an [NullAllowed] on parameter #0
 !missing-null-allowed! 'System.Void SceneKit.SCNText::set_ChamferProfile(UIKit.UIBezierPath)' is missing an [NullAllowed] on parameter #0
 !missing-null-allowed! 'System.Void SceneKit.SCNText::set_Font(UIKit.UIFont)' is missing an [NullAllowed] on parameter #0
+
+# watchOS is 32bits but our [Native] enums are all 64 bits
+!extra-enum-value! Managed value 18446744073709551615 for SCNPhysicsCollisionCategory.All not found in native headers
+!missing-enum-value! SCNPhysicsCollisionCategory native value SCNPhysicsCollisionCategoryAll = 4294967295 not bound
```

Comment:
Maybe add an exception to the enum check that int.MaxValue and long.MaxValue are acceptable synonyms for [Native] enums?

Reply:
not sure it's safe to assume 64 == 32, but it should be fine to make 32 == 64